alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -201,4 +203,49 @@ public function deleteOrMarkForDeletion($entry = null)
 		KalturaLog::debug("Live entry with id [{$liveEntry->getId()}], is set with recording disabled, clearing entry server node id [{$this->getId()}] from db");
 		$this->delete();
 	}
+
+	public function getRecordedProperties()
+	{
+		$recordedProps = $this->getFromCustomData(self::CUSTOM_DATA_RECORDED_PROPERTIES);
+		if (!$recordedProps)
+		{
+			$recordedProps = new LiveEntryServerNodeRecordedProperties();
+		}
+		return $recordedProps;
+
+	}
+
+	public function setRecordedProperties($v)
+	{
+		return $this->putInCustomData(self::CUSTOM_DATA_RECORDED_PROPERTIES, $v);
```

Comment:
You are saving an object should save services value instead of the object itself